finance_manager/src/main/resources/db/changelog/changeset/create-table-users.sql

```diff
@@ -0,0 +1,10 @@
+--liquibase formatted sql
+--changeset "gs":2
+
+CREATE TABLE IF NOT EXISTS users
+(
+    u_id       bigserial PRIMARY KEY,
+    u_email    varchar(255) NOT NULL UNIQUE,
+    u_password varchar(100) NOT NULL,
+    u_is_admin boolean      NOT NULL

```

Comment:
Roles are likely to be in ENUM

Reply:
If there is time, we can change that. But for now, let it be as it is - it's easier.